src/object_list.c

```diff
@@ -130,14 +133,14 @@ void object_create_identifier_name(char* string_buffer, const rct_object_entry*
 
 	for (uint8 i = 0; i < 4; ++i){
```

Comment:
This function should have just been replaced with `sprintf(string_buffer, "%.8s/%4X%4X", object->name, object->flags, object->checksum);`